frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "moment": "^2.29.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-hot-toast": "^2.4.0",
```

Comment:
Do you use it anywhere? I didn't see any toast getting fired?

Reply:
Tried, but didn't get it to work yet.